src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs

```diff
@@ -50,6 +50,22 @@ public class SkyHookProxy : IProvideMovieInfo, ISearchForNewMovie, IDiscoverNewM
             _logger = logger;
         }
 
+        public List<int> GetChangedMovies (DateTime startTime)
+        {
+            var request = _movieBuilder.Create()
+                .SetSegment("route", "movie")
+                .SetSegment("id", "")
+                .SetSegment("secondaryRoute", "changes")
+                .Build();
```

Comment:
Need to pass start date into the request I think

Reply:
Oops